armhf.Dockerfile

```diff
@@ -1,11 +1,15 @@
-FROM arm32v7/alpine:3.12 as frontend
+FROM alpine:3.12 as frontend
 
 RUN apk add --no-cache \
-  npm
+  npm \
+  curl
 
 RUN npm install -g @angular/cli
 
 WORKDIR /build
+
+RUN curl -L https://github.com/balena-io/qemu/releases/download/v3.0.0%2Bresin/qemu-3.0.0+resin-arm.tar.gz | tar zxvf - -C . && mv qemu-3.0.0+resin-arm/qemu-arm-static .
```

Comment:
This is a clunky weird hack which we shouldn't rely on. Unfortunately I have good device to test this all on.

Reply:
Just ordered another raspberry pi as I think I misplaced my last one lol, once that's in I'll do some testing there.